docs/jwst/introduction.rst

```diff
@@ -153,7 +153,7 @@ the ``--save-parameters <filename>`` option. For example, to save the
 configuration for the `tweakreg` step, one would do the following:
 ::
 
- strun jwst.tweakreg.PersistenceStep --save-parameters persistence.asdf
+ strun jwst.tweakreg.PersistenceStep jw00017001001_01101_00001_nrca1_uncal.fits --save-parameters persistence.asdf
```

Comment:
Sorry, but ``tweakreg`` and ``persistence`` are mutually exclusive steps. Pick one or the other.